src/restic/snapshot_filter_test.go

```diff
@@ -235,57 +164,58 @@ var expireTests = []restic.ExpirePolicy{
 	{Yearly: 10},
 	{Daily: 7, Weekly: 2, Monthly: 3, Yearly: 10},
 	{Tags: []string{"foo"}},
+	{Tags: []string{"foo", "bar"}},
 }
 
 func TestApplyPolicy(t *testing.T) {
 	for i, p := range expireTests {
-		keep, remove := restic.ApplyPolicy(testExpireSnapshots, p)
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
```

Comment:
FYI (no need to change this here): you can leave the name for the subtest empty, then the testing library will automatically assign a sequential number that can then be used for filtering subtests.